### DISCUSSION TITLE
VLANs only working in VMs but not LXC containers

### DISCUSSION THREAD

**Discussion by vulcan4d** [tteck/Proxmox#1182] [Help Needed]
I deployed a few of these template LXC containers and a couple VMs.  After that I setup my VLAN, configured the proxmox bridge to be vlan aware and tagged the VM/LXC network interfaces with the proper VLAN.  All the VMs grabbed the DHCP IPs from the new VLAN, all LXC containers did not.  I'm curious why is it different?  Many of the guides out there seem outdated for an older version of Proxmox and most have both VMs and LXC Containers not working, instead of just one and not the other.  I'm curious if you guys came accross this?

**Comment by vulcan4d** [comment]
Odd, started working after I added a second interface to the container.  Then I deleted it, added a tag on the original eth0 interface and now it works.  Weird.  Nevermind, fix now...somehow.

**Reply by emanuellopes** [reply]
I have the same issue, when I add multiples interfaces with different vlan, only one ip will work.

Very strange this issue.
